### PR TITLE
Explain block delay for operator state

### PR DESCRIPTION
**Pull request by ian-shim**

### CODE REVIEW

File: docs/eigenda/operator-guides/run-a-node/registration.mdx [hunk 1]
```diff
@@ -73,7 +73,8 @@ More information about the registration process that is executed by the above co
 
 ## Check for network traffic
 
-Once you have successfully opted into a quorum, you should begin to see logs indicating that your node is receiving, validating, and storing batches from the network, like the following:
+EigenDA uses the operator state that is 75 blocks (1.5 hours) behind the current chain head to ensure the state is not at risk of being reorg'd.
+About 1.5 hours after you have successfully opted into a quorum, you should begin to see logs indicating that your node is receiving, validating, and storing batches from the network, like the following:
```
Comment (mooselumph) [inline docs/eigenda/operator-guides/run-a-node/registration.mdx:77]: 15 minutes right?

Comment (ian-shim) [reply docs/eigenda/operator-guides/run-a-node/registration.mdx:77]: 🤦

**Review by mooselumph — APPROVED** [review]

File: docs/eigenda/operator-guides/run-a-node/registration.mdx [hunk 2]
```diff
@@ -73,7 +73,8 @@ More information about the registration process that is executed by the above co
 
 ## Check for network traffic
 
-Once you have successfully opted into a quorum, you should begin to see logs indicating that your node is receiving, validating, and storing batches from the network, like the following:
+EigenDA uses the operator state that is 75 blocks (15 minutes) behind the current chain head to ensure the state is not at risk of being reorg'd.
```
Comment (mooselumph) [inline docs/eigenda/operator-guides/run-a-node/registration.mdx:76]: Having the parameter hard-coded into the doc probably isn't ideal I guess. But this seems fine for now, we can clean up later.

Comment (ian-shim) [reply docs/eigenda/operator-guides/run-a-node/registration.mdx:76]: +1 I don't know if there's a way to load these parameters. 
We also have avssync interval (6h) hardcoded in L60

**Pull request merged**